### PR TITLE
Display email+comments

### PR DESCRIPTION
**Pull request by olu144**

### CODE REVIEW

**Review by dslucas — COMMENTED** [review]
Could you try merging from master one more time? It seems like some of the code changes from your previous PRs got overwritten or duplicated. It might be helpful to take another pass at the code and make sure everything looks correct. I know it's a pain :-/

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -40,21 +40,26 @@ function addRandomStock() {
...
-function LoadComments() {
-  const max = getMax();
-  fetch('/data?numCommentsToLoad=' + max).then(response => response.json()).then((comments) => {
-  const commentListElement = document.getElementById('comment-list');
-  commentListElement.innerHTML = ' ';
-  comments.forEach((comment) => {
-  commentListElement.appendChild(createListElement(comment));
+async function LoadComments() {
+  const loggedIn = await isLoggedIn();
+  if(loggedIn) {
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: small nit, there should be a space after `if`

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -40,21 +40,26 @@ function addRandomStock() {
...
+    const max = getMax();
+    fetch('/data?numCommentsToLoad=' + max).then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comment-list');
+    commentListElement.innerHTML = ' ';
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createListElement(comment));
+      });
     });
-  });
+  }else{
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: also small nit: there should be spaces around `} else {`

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -40,21 +40,26 @@ function addRandomStock() {
...
-  commentListElement.innerHTML = ' ';
-  comments.forEach((comment) => {
-  commentListElement.appendChild(createListElement(comment));
+async function LoadComments() {
+  const loggedIn = await isLoggedIn();
+  if(loggedIn) {
+    const max = getMax();
+    fetch('/data?numCommentsToLoad=' + max).then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comment-list');
+    commentListElement.innerHTML = ' ';
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: tiny nit: this is usually just '', rather than ' ' (should be no space)

**Review by dborkan — COMMENTED** [review]
New code LGTM (looks good to me), once conflicts are resolved

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (olu144) [reply portfolio/src/main/webapp/script.js]: addressed

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (olu144) [reply portfolio/src/main/webapp/script.js]: addressed in "address nits"

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (olu144) [reply portfolio/src/main/webapp/script.js]: addressed in "address nits"

**Review by dslucas — APPROVED** [review]

**Pull request merged**